### ISSUE TITLE
GonzaNahue

### ISSUE THREAD

**Issue by GonzaNahue** [rafaelvcaetano/melonDS-android#177]
* **Device model:** Lenovo A6020l37
* **Android version:** Android API 22

**Problem:**  
Insert a small description of the problem.

**Stack trace:**  
```
me.magnum.melonds.ui.emulator.EmulatorActivity$f: Failed to load ROM: NDS_FAILED
	at me.magnum.melonds.ui.emulator.j$c$a$a.a()
	at f.a.b0.e.e.a.m()
	at f.a.s.a()
	at f.a.b0.e.b.e$a.d()
	at f.a.b0.e.b.l$a.d()
	at f.a.b0.e.b.j$a.run()
	at f.a.r$a.run()
	at f.a.b0.g.j.run()
	at f.a.b0.g.j.call()
	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:152)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:265)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
	at java.lang.Thread.run(Thread.java:818)

```

**Comment by mdshamsulhaque** [comment]
Can you use the latest beta [v1.3.0](https://github.com/rafaelvcaetano/melonDS-android/releases/tag/beta-1.3.0) cnd open a new issue? The latest version does not fix the problem but includes additional info that will help to find the cause.

**Comment by shabbirmahmud** [comment]
Can you use the latest beta [v1.3.0](https://github.com/rafaelvcaetano/melonDS-android/releases/tag/beta-1.3.0) and open a new issue? The latest version does not fix the problem but includes additional info that will help to find the cause.

**Comment by rafaelvcaetano** [comment]
If your problem is not yet fixed, try the latest [Beta 1.4.0](https://github.com/rafaelvcaetano/melonDS-android/releases/latest) release. If the problem is not fixed, please open a new issue.
